swarm/api/manifest.go

```diff
@@ -33,7 +33,8 @@ import (
 )
 
 const (
-	ManifestType = "application/bzz-manifest+json"
+	ManifestType   = "application/bzz-manifest+json"
+	DbManifestType = "application/bzz-db-manifest+json"
```

Comment:
Why is this a new mime type? What is different about it?

Reply:
Hm, good question. First I was thinking that since this is not a "file" but more of a database entry, the `Mode` doesn't mean exactly the same as it does when used in a file system. But 1) maybe it does and 2) ordinary swarm manifests don't really treat content as "files" anyway.

Reply:
> since this is not a "file" but more of a database entry

I'm not sure I follow, what makes a resource update of a manifest any different to a normal update? Perhaps I am missing what was implemented in the resource updates.